front/app/containers/ProjectsShowPage/shared/survey/TypeformSurvey.tsx

```diff
@@ -70,10 +66,8 @@ const TypeformSurvey = memo<Props>(
         <Iframe
           url={surveyUrl}
           width="100%"
-          height={
-            smallerThanLargeTablet ? surveyHeightMobile : surveyHeightDesktop
-          }
-          display={isIframeLoaded ? 'block' : 'none'}
+          height={isLargeTablet ? surveyHeightMobile : surveyHeightDesktop}
+          styles={{ visibility: isIframeLoaded ? 'visible' : 'hidden' }}
```

Comment:
`display: none` was breaking the Typeform display on mobile for whatever reason so I switched it to `visibility: hidden`